### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by nfriedly**
Update the urls and add @trivikr to the contributors list.

Feel free to add a URL or email if you'd like.

### CODE REVIEW

**Comment by nfriedly** [comment]
Also, if you have an npm acount @trivikr, let me know your username there and I'll grant you publish permission so that CI isn't the *only* option for you. (Although I still generally recommend letting CI do the publishing.)

File: package.json [hunk 1]
```diff
@@ -36,13 +36,16 @@
     {
       "name": "Nathan Friedly",
       "url": "http://nfriedly.com"
+    },
+    {
+      "name": "Trivikram Kamat"
```
Comment (trivikr) [inline package.json]:
```suggestion
      "name": "Trivikram Kamat",
      "url": "https://twitter.com/trivikram"
```

**Comment by trivikr** [comment]
Thank you, @nfriedly, for adding me as a contributor.

> Also, if you have an npm acount @trivikr, let me know your username there

My npm username is [trivikr](https://www.npmjs.com/~trivikr)

**Review by trivikr — APPROVED** [review]

**Comment by nfriedly** [comment]
Awesome, you are now listed as an owner/contributor on https://www.npmjs.com/package/microphone-stream

**Pull request merged**